packages/diffhtml-components/lib/render-component.js

```diff
@@ -106,7 +106,7 @@ export default function renderComponent(vTree, transaction) {
        * @param {any} state
        */
       render(props, state) {
-        return createTree(RawComponent(props, state));
+        return createTree(vTree.rawNodeName(props, state));
```

Comment:
This seems like an unnecessary change. Possibly there was an inconsistency that led to me writing this. Should try and produce a unit test that shows a failure if I revert this.